public/app/plugins/panel/geomap/components/DataHoverView.tsx

```diff
@@ -59,56 +62,92 @@ export const DataHoverView = ({ data, rowIndex, columnIndex, sortOrder, mode }:
   }
 
   return (
-    <table className={styles.infoWrap}>
-      <tbody>
-        {(mode === TooltipDisplayMode.Multi || mode == null) &&
-          displayValues.map((v, i) => (
-            <tr key={`${i}/${rowIndex}`} className={i === columnIndex ? styles.highlight : ''}>
-              <th>{v[0]}:</th>
-              <td>{v[2]}</td>
+    <div className={styles.wrapper}>
+      <div className={styles.header}>
+        <span className={styles.title}>Exemplar</span>
+      </div>
```

Comment:
I think if we make this header conditional / opt out via a passed in prop (can default to true) we can remove this from geomap tooltip (where it might not make as much sense) and be good to merge 👍